pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
```

Comment:
Generated by my `pipenv lock`

Reply:
i don't think we need this in github

Reply:
It looks like, if I use a modern `pipenv`, it automatically re-generates the file. Both`pipenv install` and `pipenv lock` re-recreate it. I'm thinking it may provide context to the installer and maybe is worth keeping here?